http/param/param_test.go

```diff
@@ -451,6 +451,43 @@ func TestParser_Parse_PathParam_FuncNotDefinedError(t *testing.T) {
 	assert.Error(t, err)
 }
 
+type structWithFormParams struct {
+	Subject string            `param:"form=subject"`
+	Amount  *int              `param:"form=amount"`
+	Object  *maybeShinyObject `param:"form=object"`
```

Comment:
consider also a test where optional (pointer) fields are not present in the request (similar to `TestParser_Parse_QueryParam_Pointers` - `no params`